### PR TITLE
[AC-2436] Show unassigned items banner in browser

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Extend #8655 to browser.

It's stacked on #8655 as the web vault is the main priority, this can be reviewed and merged separately (after that one).

Notes:
* I used the same wording, however there's less space here so I've asked product to revise it as it's a bit wordy
* this replaces the autofill banner. As with the web vault and the UI banner, I've just replaced it in the template, the team responsible need to clean up the rest of the (now unused) banner code.
* otherwise, this uses the same service and logic as web, so what works there should work here.

## Code changes

- **file.ext:** Description of what was changed and why

## Screenshots
Screenshot 2024-04-10 at 2 29 35 PM

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/browser/src/_locales/en/messages.json [hunk 1]
```diff
@@ -3005,5 +3005,8 @@
   },
   "passkeyRemoved": {
     "message": "Passkey removed"
+  },
+  "unassignedItemsBanner": {
+    "message": "Unassigned organization items are no longer visible in your All Vaults view across devices and are now only accessible via the Admin Console. Assign these items to a collection from the Admin Console to make them visible."
```
Comment (addisonbeck) [inline apps/browser/src/_locales/en/messages.json:3010]: Copy change from product:
```suggestion
    "message": "Notice: Unassigned organization items are no longer visible in your All Vaults view across devices and are now only accessible via the Admin Console. Assign these items to a collection from the Admin Console to make them visible."
```

**Review by gbubemismith — APPROVED** [review]
Looks good for team vault.

**Comment by gbubemismith** [comment]
Looks like there are some linting issues

**Review by addisonbeck — APPROVED** [review]

**Review by shane-melton — APPROVED** [review]
